test/FunctionalTests/Microsoft.OData.Core.Tests/Build.NetFramework/Microsoft.OData.Core.Tests.csproj

```diff
@@ -453,6 +453,8 @@
     <Compile Include="..\WriterUtilsTests.cs" />
     <Compile Include="..\ScenarioTests\Streaming\ODataJsonLightStreamReadingTests.cs" />
     <Compile Include="..\ScenarioTests\Streaming\ODataJsonLightStreamWritingTests.cs" />
+    <Compile Include="TestMetadataSelector.cs" />
+    <Compile Include="ODataMetadataSelectorTests.cs" />

```

Comment:
These seem to be in the wrong location.  Other tests are one level up (for common/core tests), or under "ScenarioTests" or other categories.  These should be under "JsonLight" (i.e.., ..\JsonLight\TestMetadataSelector.cs).